dotCMS/src/main/java/com/dotcms/rest/BundleResource.java

```diff
@@ -218,4 +240,295 @@ public Response deleteEnvironmentPushHistory(@Context HttpServletRequest request
         return responseResource.response( "true" );
 	}
 
+    /**
+     * Deletes all bundles by identifier
+     * Note: the response will be notified by socket message
+     * @param request   {@link HttpServletRequest}
+     * @param response  {@link HttpServletResponse}
+     * @param deleteBundlesByIdentifierForm {@link DeleteBundlesByIdentifierForm} contains the set of bundle ids to delete.
+     */
+	@DELETE
+    @Path("/ids")
```

Comment:
This should not be needed in the path. I see two options here:
1. Send a comma separated list of ids. `DELETE /bundles/1,2,3` 
2. Send a body with the Ids to deletee